.travis.yml

```diff
@@ -20,21 +20,23 @@ addons:
 
 jobs:
   include:
-    - name: lints
-        install:
-          - pip install black
-          - pip install pylint
-          - pip install mypy
-          - pip install --no-binary=protobuf protobuf pysodium requests websocket-client-py3 PyYAML
-          - git clone https://github.com/exonum/exonum-python-client.git
-          - pip install -e exonum-python-client
-        script:
-          - black --check -l 120 . --exclude=".*_pb2.py" --exclude="exonum-python-client"
-          - mypy --ignore-missing-imports --disallow-untyped-defs ./exonum_launcher
-          - pylint exonum_launcher --max-line-length=120 --disable=fixme,bad-continuation,too-few-public-methods,duplicate-code
-    - name: tests
-        install:
-          - pip install --no-binary=protobuf protobuf pysodium requests websocket-client-py3 PyYAML
-          - git clone https://github.com/exonum/exonum-python-client.git
-          - pip install -e exonum-python-client
-        script: python -m unittest -v
+  - name: lints
+    install:
+      - pip install black
+      - pip install pylint
+      - pip install mypy
+      - pip install --no-binary=protobuf protobuf pysodium requests websocket-client-py3 PyYAML
+      - git clone https://github.com/exonum/exonum-python-client.git
+      - pip install -e exonum-python-client
+    script:
+      - black --check -l 120 . --exclude=".*_pb2.py" --exclude="exonum-python-client"
+      - mypy --ignore-missing-imports --disallow-untyped-defs ./exonum_launcher
+      - pylint exonum_launcher --max-line-length=120 --disable=fixme,bad-continuation,too-few-public-methods,duplicate-code
+  - name: tests
+    install:
+      - pip install --no-binary=protobuf protobuf pysodium requests websocket-client-py3 PyYAML
+      - git clone https://github.com/exonum/exonum-python-client.git
+      - pip install -e exonum-python-client
+    script:
+      - echo "foo"
```

Comment:
Why?

Reply:
Something weird happens with the Travis 😕 Just trying to figure out.